js/react_native/e2e/e2e/OnnxruntimeModuleExample.test.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/* Initial Simple Detox Test Setup. Can potentially add more unit tests. */
+
+describe('OnnxruntimeModuleExample', () => {
+  beforeAll(async () => {
+    await device.launchApp();
+  });
+
+  beforeEach(async () => {
+    await device.launchApp({ newInstance: true });
+  });
+
+  it('OnnxruntimeModuleExampleE2ETest CheckOutputComponentExists', async () => {
+    await element(by.label('output'));
+  });
+
+  it('OnnxruntimeModuleExampleE2ETest CheckInferenceResultValueIsCorrect', async () => {
+    if (device.getPlatform() === 'ios') {
+      await expect(element(by.label('output')).atIndex(1)).toHaveText('Result: 3');
```

Comment:
on platform ios, if not specified index 1 will run into `multiple labels found` compiling error.

Reply:
just curious, what are the other elements with label "output"?

Reply:
not exactly sure, but I checked the debugging logs, it might be some label elements from the underlying implementations.
only on iOS though)

Reply:
also another way suggested by Detox would be to use unique id for UI elements to avoid using `atIndex()`. https://wix.github.io/Detox/docs/api/matchers/#atindex will take a look

Reply:
any luck with using a unique ID? it would be nice to have platform independent test code

Reply:
yeah, I agree. so I tried to use testID to replace the accessibilityLabel currently used  for the `Result:3` element in the app, seems like android side detox tests complained about not finding this UI element but ios passed though. so figured might just leave it like this for now as the CIs are passing.

Reply:
ok. an alternative idea - can we just use a more specifically named accessibilityLabel value? e.g., "inferenceResult". maybe we can get a single result then